VogtPayroll2/Employee1.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll2
+{
+    abstract class Employee
+    {
+        //For simplicity, use a constant tax rate of 30% to compute the tax amount
+        protected string name;
+        protected int hoursWorked;
+        protected decimal payRate;
```

Comment:
Get rid of the `payRate` here. Instead, use the abstract `GetPayRate()` function.